benchmark/parse/filter.benchmark.cpp

```diff
@@ -14,7 +14,8 @@ using namespace mbgl;
 style::Filter parse(const char* expression) {
     rapidjson::GenericDocument<rapidjson::UTF8<>, rapidjson::CrtAllocator> doc;
     doc.Parse<0>(expression);
-    return *style::conversion::convert<style::Filter, JSValue>(doc);
+    style::conversion::Error error;
+    return *style::conversion::convert<style::Filter, JSValue>(doc, error);
```

Comment:
This produces a 💥 if conversion fails. Given that this is a test case, can we just add a `ASSERT_TRUE`, which fails hard when there is no conversion result?